src/site/stages/build/drupal/graphql/eventListingPage.graphql.js

```diff
@@ -11,6 +11,42 @@ module.exports = `
     title
     fieldIntroText
     entityId
+    secondSetOfEvents: reverseFieldListingNode(limit: 500, filter: {conditions: [{field: "status", value: "1", operator: EQUAL}, {field: "type", value: "event"}]}, sort: {field: "changed", direction: DESC}) {
```

Comment:
let's rename this query to indicate that these are past events

Reply:
disregard this comment, I see how this query is used now 👍

Reply:
:+1: @oksana-c